redoc.json

```diff
@@ -194,6 +194,36 @@
 			"repo": "reaction-docs",
 			"docPath": "developer/data-fixtures-insert-product-images.md"
 		}, {
+      "class": "guide-nav-item",
+      "alias": "swag-shop-tutorial",
+      "label": "Swag shop tutorial",
+      "repo": "reaction-docs",
+      "docPath": "developer/swag-shop/introduction.md"
+    }, {
+      "class": "guide-sub-nav-item",
+      "alias": "swag-shop-collecting-requirements",
+      "label": "Collecting the shop requirements",
+      "repo": "reaction-docs",
+      "docPath": "developer/swag-shop/swag-shop-collecting-requirements.md"
+    }, {
+      "class": "guide-sub-nav-item",
+      "alias": "swag-shop-initialization",
+      "label": "Project scaffolding",
+      "repo": "reaction-docs",
+      "docPath": "developer/swag-shop/swag-shop-initilization.md"
```

Comment:
`docPath` should be `developer/swag-shop/swag-shop-initialization.md`

Reply:
Let's also number these posts like:

Part 1: Collecting the shop requirements
Part 2: Project scaffolding
...

Reply:
done